src/main.rs

```diff
@@ -41,3 +43,9 @@ fn test_morning_greeting() {
     let greeter = Greeter::new("Test".into());
     assert!(greeter.get_greeting(6).contains("morning"));
 }
+
+#[test]
+fn test_day_greeting() {
+    let greeter = Greeter::new("Test".into());
+    assert!(greeter.get_greeting(15).contains("day"));
```

Comment:
14 is a better number
```suggestion
    assert!(greeter.get_greeting(14).contains("day"));
```